src/components/departure-list/DeparturesList.tsx

```diff
@@ -81,6 +81,7 @@ export default function NearbyResults({
               currentLocation={currentLocation}
               lastUpdated={lastUpdated}
               defaultExpanded={i === 0}
+              disableCollapsing={disableCollapsing}
```

Comment:
This also seems to hide headers altogether. Should it be named something more accurate to reflect that?

Reply:
It hides the collapsible tool header. 

![scrn223](https://user-images.githubusercontent.com/1464594/113920793-8cc5a480-97e5-11eb-8dce-714434462dbe.png)

it seemed unnecessary when collapsing was disabled. 
Do you think we should still show it?

Reply:
@pettersuul maybe?

Reply:
I was thinking more like renaming `disableCollapsing` to be `hideHeader` for instance (or something similar ofc)

Reply:
On second thought, this will show several quays right? Not one specific quay, but one stop place? In that case we must show the header as it can be named platforms like `Lerkendal 1`, `Lerkendal 2` etc.

Reply:
Ah, yes, In the specific view where disable collapsing is actually used (quay departures), we already show the stop name in the main header.
So this feature was introduced to remove some of the elements that would be perceived as unnecessary clutter.

The option to disable collapsing is used in this view because we know we are only showing platforms for one stop, and the stop name is already displayed in the main header. In this specific case there is no need to collapse platforms to save space, nor show the extra header with the stop name.

If we are showing a view where it makes sense to group platforms together in a stop, we should definitely not be using the  disable collapsing option. 

In the case when we are showing departures for a specific stop, there are two headers showing the stop name and 1 showing the platform name before the actual departures are listed, this removes one of the stop name headers.



Of course, this is my hypothesis. user testing might well prove me wrong.

Reply:
![image](https://user-images.githubusercontent.com/1464594/114167987-f225ac80-992f-11eb-81a8-81a999d526e8.png)
vs
![image](https://user-images.githubusercontent.com/1464594/114168206-3c0e9280-9930-11eb-9a17-91d3bd67c565.png)

Reply:
collapses to
![image](https://user-images.githubusercontent.com/1464594/114168314-62ccc900-9930-11eb-8e45-872ecfe550fe.png)

Reply:
If only one stop is listed (e.g. searched for å stop in departures), the collapsible stop-header is unneccesary.

![image](https://user-images.githubusercontent.com/56530785/114372842-4aec8380-9b82-11eb-80e2-7d53222add02.png)